twig/report/policy/error.html.twig

```diff
@@ -1,17 +1,24 @@
 
-<div class="row result-group">
-  <div class="col-sm-12 col-md-10">
-    <h4 id="{{ audit_response.policy.name }}">{{ audit_response.policy.title }} <small class="label label-{{ audit_response.type }}">{{ audit_response.type|split('-')|join(' ')|title  }} <span class="glyphicon {% if audit_response.isSuccessful %}glyphicon-ok{% else %}glyphicon-remove{% endif %}" aria-hidden="true"></span></small></h4>
+<div class="row result-group py-2">
+  <div class="col">
+    <h4 id="{{ audit_response.policy.name }}"><span class="text-danger">&#9785;</span> {{ audit_response.policy.title }}</h4>
```

Comment:
audit_response.policy.name may not be an HTML compliant ID. I just checked and it doesn't look like Twig has a built in fix for this (like a html_id filter or something). I don't think this should stop the PR, but just want to call out we might want to add such a filter to Drutiny at some point.